_editorial_policies/02_editor_instructions.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: single
+sidebar:
+  nav: editor_instructions.md
+title: Instructions for Editors
+excerpt: Instructions for editors
+permalink: /policies/editor_instructions/
+---
+
+This provides instructions to editors handling manuscripts for LiveCoMS.
+
+## Workflow overview
+
+Authors contact LiveCoMS (specifically, the Lead Editor in a particular subject area) with a presubmission letter, proposing an article in a particular area.
+Once this is approved, they proceed with article preparation and ultimately submit it to that section of LiveCoMS.
+The Lead Editor then checks that the article is consistent with what was proposed in the presubmission letter, and then assigns it to an appropriate Associate Editor who manages the review process and makes a final decision on the manuscript, reporting the final decision to the authors.
+
+## Pre-review processing
+
+### Lead Editor
+
+The Lead Editor:
+- Handles analysis of presubmission inquiries
+- On article submission, confirms that the article is consistent with the presubmission letter
+- Passes the article and presubmission letter to an appropriate Associate Editor to manage the review process (unless he or she decides to manage the review process directly)
+- Makes a final decision after receiving a recommendation from the Associate Editor and reviewers (see below)
+
+### Associate Editor
+
+Before assigning manuscripts for review, editors have several main tasks:
+- Ensure they do not have a conflict of interest with respect to the work they are to analyze; if they do, [dealing with as dictated by editorial policy](https://livecomsjournal.github.io/policies/editorial_board/).
+- Check to ensure that the manuscript has appropropriate style, grammar, layout, and figure quality to be ready for editing, as in the [instructions for authors](https://livecomsjournal.github.io/authors/policies/). Remember, the journal will not be editing the manuscript, so if you will need to reject the manuscript (for additional revision) because of grammar issues or other stylistic reasons, you should do this *before* sending it for review to avoid wasting the time of the reviewers.
+- Identify suitable reviewers, who may include experts suggested by the authors, others in the field you already know of, or authors cited frequently in the article. LiveCoMS generally requires at least two reviewers, though exceptional circumstances (such as extensive community feedback via GitHub) may result in exceptions.
```

Comment:
Something very important: Previously we had committed to having one student reviewer per paper.  That should be indicated here.  It's also something we might use in our promotional materials.  It's rather unique!

Reply:
Agreed. Michael merged this; do a PR to add this?